### PR TITLE
Update documentation for the release v0.6.2

### PR DESCRIPTION
**Pull request by xmudrii**
**What this PR does / why we need it**:

Update documentation for the release v0.6.2 which is going to be released today.

**Release note**:
```release-note
NONE
```

/assign @kron4eg

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# [v0.6.2](https://github.com/kubermatic/kubeone/releases/tag/v0.6.2) - 2019-05-13
+
+## Changed
+
+* Add a missing JSON tag on the `Name` field, so specifying `name` in lowercase in the manifest works as expected ([#439](https://github.com/kubermatic/kubeone/pull/439))
```
Comment (kron4eg) [inline CHANGELOG.md]:
```suggestion
* Fix missing JSON tag on the `Name` field, so specifying `name` in lowercase in the manifest works as expected ([#439](https://github.com/kubermatic/kubeone/pull/439))
```

**Comment by kron4eg** [comment]
/lgtm
/approve

/hold
hold until actual release

**Comment by xmudrii** [comment]
/override pull-kubeone-e2e-digitalocean-conformance-1.14.1
/override pull-kubeone-e2e-digitalocean-upgrade-1.13.5-1.14.1
/override pull-kubeone-e2e-digitalocean-conformance-1.13.5

**Comment by xmudrii** [comment]
/override pull-kubeone-e2e-digitalocean-upgrade-1.13.5-1.14.1

**Comment by xmudrii** [comment]
Docs change only.
/override pull-kubeone-e2e-aws-conformance-1.13.5
/override pull-kubeone-e2e-aws-conformance-1.14.1
/override pull-kubeone-e2e-digitalocean-conformance-1.13.5
/override pull-kubeone-e2e-digitalocean-conformance-1.14.1
/override pull-kubeone-e2e-aws-upgrade-1.13.5-1.14.1
/override pull-kubeone-e2e-digitalocean-upgrade-1.13.5-1.14.1

**Pull request merged**